ml4gw/waveforms/taylorf2.py

```diff
@@ -69,9 +122,69 @@ def taylorf2_phase(
         - eta * eta * eta * 127.825 / 1.296
         + pfa_v6logv * torch.log(torch.tensor(4.0))
     )
+    # SO + S1-S2 + S-squared contribution at 3 PM
```

Comment:
Minor comment that I won't hold this up for: this comment should say `3PN` rather than `3PM`